build-tools/automation/yaml-templates/commercial-build.yaml

```diff
@@ -66,62 +66,6 @@ steps:
   workingDirectory: ${{ parameters.xaSourcePath }}
   displayName: make all-tests
 
-# Restore needs to be executed first or MicroBuild targets won't be imported in time
-- task: MSBuild@1
-  displayName: msbuild /t:Restore sign-content.proj
-  inputs:
-    solution: ${{ parameters.xaSourcePath }}/build-tools/installers/sign-content.proj
-    configuration: $(XA.Build.Configuration)
-    msbuildArguments: /t:Restore /bl:${{ parameters.xaSourcePath }}/bin/Build$(XA.Build.Configuration)/restore-sign-content.binlog
-
-- task: MSBuild@1
-  displayName: PKG signing - add entitlements and sign classic libraries
-  inputs:
-    solution: ${{ parameters.xaSourcePath }}/build-tools/installers/sign-content.proj
-    configuration: $(XA.Build.Configuration)
-    msbuildArguments: >-
-      /t:AddMachOEntitlements;AddMSBuildFilesUnixSign;Build
-      /p:SignType=$(MicroBuildSignType)
-      /p:MicroBuildOverridePluginDirectory=$(Build.StagingDirectory)/MicroBuild/Plugins
-      /bl:${{ parameters.xaSourcePath }}/bin/Build$(XA.Build.Configuration)/sign-content.binlog
-  condition: and(succeeded(), eq('${{ parameters.signClassicPkgContent }}', 'true'))
-
-- task: MSBuild@1
-  displayName: PKG signing - sign classic executables
-  inputs:
-    solution: ${{ parameters.xaSourcePath }}/build-tools/installers/sign-content.proj
-    configuration: $(XA.Build.Configuration)
-    msbuildArguments: >-
-      /t:AddMSBuildFilesUnixSignAndHarden;Build
-      /p:SignType=$(MicroBuildSignType)
-      /p:MicroBuildOverridePluginDirectory=$(Build.StagingDirectory)/MicroBuild/Plugins
-      /bl:${{ parameters.xaSourcePath }}/bin/Build$(XA.Build.Configuration)/sign-content.binlog
-  condition: and(succeeded(), eq('${{ parameters.signClassicPkgContent }}', 'true'))
-
-- task: MSBuild@1
-  displayName: PKG signing - sign binutils libraries
-  inputs:
-    solution: ${{ parameters.xaSourcePath }}/build-tools/installers/sign-content.proj
-    configuration: $(XA.Build.Configuration)
-    msbuildArguments: >-
-      /t:AddBinUtilsFilesUnixSign;Build
-      /p:SignType=$(MicroBuildSignType)
-      /p:MicroBuildOverridePluginDirectory=$(Build.StagingDirectory)/MicroBuild/Plugins
-      /bl:${{ parameters.xaSourcePath }}/bin/Build$(XA.Build.Configuration)/sign-content.binlog
-  condition: and(succeeded(), eq('${{ parameters.signClassicPkgContent }}', 'true'))
-
-- task: MSBuild@1
-  displayName: PKG signing - sign binutils executables
```

Comment:
I'll need to take a closer look at this, I believe we'll still need to sign macOS specific files that go into our .NET NuGet packages. We likely won't be able to remove all of these steps.

Reply:
We can leave them alone if we don't want to tackle this right away.  I only removed them because the naming suggested they were for Classic only.

Reply:
Here's a diff that should only remove classic file MacDev cert signing https://gist.github.com/pjcollins/28b1bd5341ffc7a09a16b9d71436e98f. We'll also want to manually queue an official build against this branch to see if anything else breaks

Reply:
Applied the patch.  Looks like the SBOM job is failing on the "main" pipeline:
https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=7554443&view=results

Feel free to commit fix(es) to this branch if needed.

Reply:
Looks like it's fixed now:
https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=7581927&view=results